proto/spire/agent/nodeattestor/v0/nodeattestor.go

```diff
@@ -0,0 +1,95 @@
+// Provides interfaces and adapters for the NodeAttestor service
```

Comment:
This is just the moved plugin scaffolding codegen. This will change again with the new catalog system.